### DISCUSSION TITLE
WheelFileValidationError with furo package only on the poetry==1.4.1

### DISCUSSION THREAD

**Discussion by rbebb** [pradyunsg/furo#639] [Potential Issue] [answered]
Copied from an issue reported on Poetry's GitHub repo that was then closed in favor of bringing the discussion here. It seems that there is a WheelFileValidationError when installing furo using poetry 1.4.1, but not 1.4.0. I'm wondering if it's caused by an incorrect hash in the latest version of furo that could be fixed by updating the latest wheel for furo.

Original issue link: https://github.com/python-poetry/poetry/issues/7691#issue-1632193622

**Comment by pradyunsg** [comment]
Ah, yes.

**Reply by rbebb** [reply]
@pradyunsg Is there an update on this fix?

**Comment by alecandido** [comment]
Is this related to https://github.com/pradyunsg/sphinx-theme-builder/issues/13?

**Reply by onerandomusername** [reply]
Yeah

**Comment by pradyunsg** [comment]
This should be resolved with the release I just cut right now. :)

**Reply by doyled-it** [reply]
I'm still getting the following:
```
$ poetry add --no-cache furo@2023.03.23

Updating dependencies
Resolving dependencies... Downloading https://files.pythonhosted.org/packages/d0/4a/5677dc12d2b0c2f9fa901cc43b3f6b8d99ddac00fcfca93e9989bca1d93c/numpy-1.22.4-cp310-cp310

...

Writing lock file

Package operations: 1 install, 0 updates, 0 removals

  • Installing furo (2023.3.23): Failed

  _WheelFileValidationError

...

"In $HOME/.cache/pypoetry/artifacts/72/cc/1d/8642450cf665799ef406563c72da29cb1627134cd7a94bd974449c9506/furo-2023.3.23-py3-none-any.whl, hash / size of furo-2023.3.23.dist-info/entry_points.txt didn't match RECORD"
```
Not just for `entry_points.txt`, but seemingly all files.

**Reply by pradyunsg** [reply]
Huh, I guess time for me to check what I did wrong then.

**Comment by relsqui** [comment]
I'm still seeing this error with the 2023.03.23 version (as it looks like @doyled-it also was in that output).

**Comment by martinmiglio** [comment]
Getting error in 2023.03.23 version, poetry output in [gist here](https://gist.github.com/marmig0404/c9b3344f40ff91845a7056bff8ce6ac4)

**Reply by martinmiglio** [reply]
should this get upgraded to an issue? seems like its happening with consistency.

**Reply by mjt91** [reply]
Guess so, I am getting the same in my projects.

**Reply by faggionluca** [reply]
yup same validation error here too

**Comment by pradyunsg** [comment]
Can someone confirm that the latest release (2023.03.27) fixes this?

**Reply by kdeldycke** [reply]
Can confirm `2023.03.27` fix this issue.

**Reply by mjt91** [reply]
Same for me

**Comment by pradyunsg** [comment, marked as answer]
Alrighty. What happened here is sphinx-theme-builder was generating the wrong hashes and fixing Furo's hashes required me to cut a release of sphinx-theme-builder _then_ cut a release of Furo. I'd assumed that I'd cut a release of sphinx-theme-builder already, but hadn't. Once that was fixed there and a release was made, Furo merely needed a rebuild and release -- which is what 2023.03.27 is.

Thanks for flagging this! :)
